### PR TITLE
Custom Secondary and Package Manager implementations

### PR DESCRIPTION
**Pull request by eu-siemann**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,7 +1,12 @@
+project(libaktualizr-demo-app)
 cmake_minimum_required(VERSION 3.8)
 
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/aktualizr/include)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/aktualizr/src/libaktualizr)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/aktualizr/third_party/jsoncpp/include)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/aktualizr/third_party/googletest/googletest/include)
+
+set(AKTUALIZR_VERSION "2020.8")
```
Comment (pattivacek) [inline CMakeLists.txt:9]: Why is this necessary?

**Review by pattivacek — APPROVED** [review]
Looks good, thanks!

File: CMakeLists.txt (hunk 1 above)
Comment (eu-siemann) [reply CMakeLists.txt:9]: Yeah, that's ugly. I was getting this
```
fatal: No names found, cannot describe anything.
CMake Error at aktualizr/CMakeLists.txt:106 (message):
Could not get current version from git
```
on the CI, and it seemed the easiest way to fix it.

Comment (eu-siemann) [reply CMakeLists.txt:9]: I think I was fixing something similar before, but can't remember the details.

Comment (pattivacek) [reply CMakeLists.txt:9]: Interesting. Looks more like a libaktualizr issue. Thanks for checking!
